core/controllers/base_test.py

```diff
@@ -87,6 +87,18 @@ def test_requests_for_invalid_paths(self):
         response = self.testapp.put('/gallery/extra', {}, expect_errors=True)
         self.assertEqual(response.status_int, 404)
 
+    def test_redirect_in_both_logged_in_and_logged_out_states(self):
+        "Test for a redirect in both logged in and logged out states on '/'."
+
+        # Logged out state
+        response = self.testapp.get('/')
+        self.assertEqual(response.status_int, 302)
```

Comment:
Excellent, I'm happy it's useful! Let me know when you've submitted a new commit, and I'll take another look.